resources/asciidoctor/lib/elastic_compat_preprocessor/extension.rb

```diff
@@ -7,15 +7,23 @@
 #
 # Turns
 #   added[6.0.0-beta1]
+#   changed[6.0.0-beta1]
```

Comment:
The code and comments disagree, the code says `coming` but the comments say `changed`

Reply:
I've pushed a fix!